src/component/Header/header.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  querySearch: string,
+  setQuerySearch: (query: string) => void,
+  handleAddTodo: () => void,
+  selectAllTodos: () => void,
+  isAllTodosCompleted: boolean,
+  hasSomeTodos: boolean
+};
+
+export const Header: React.FC<Props> = ({
+  querySearch,
+  setQuerySearch,
+  handleAddTodo,
+  selectAllTodos,
+  isAllTodosCompleted,
+  hasSomeTodos,
+}) => {
+  const handleSubmit = (event: React.SyntheticEvent) => {
+    event.preventDefault();
+    handleAddTodo();
+    setQuerySearch('');
+  };
+
+  return (
+    <header className="todoapp__header">
+      {/* eslint-disable-next-line */}
+      {hasSomeTodos && <button
+        type="button"
+        className={classNames(
+          'todoapp__toggle-all',
+          { active: isAllTodosCompleted },
+        )}
+        onClick={selectAllTodos}
+      />}
```

Comment:
```suggestion
      {hasSomeTodos && (
        <button
          type="button"
          className={classNames(
            'todoapp__toggle-all',
            { active: isAllTodosCompleted },
          )}
          onClick={selectAllTodos}
          aria-label="toggle all"
        />
      )}
```
fix indentation and set aria-lable instead disable eslint